WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1695,8 +1695,8 @@ public void onPostSettingsFragmentPositiveButtonClicked(@NonNull PostSettingsLis
         }
     }
 
-    public interface AfterSavePostListener {
-        void onPostSave();
+    public interface OnPostUpdatedWithUIListener {
```

Comment:
I think this is totally fine, but I believe `OnPostUpdatedFromUIListener` would be more grammatically correct.

Reply:
👍  Fixed in 251b4056effd89f2c2c07793b5e340dcbf0ad953